base-theme/layouts/partials/footer.html

```diff
@@ -1,2 +1,53 @@
-<footer>
+<footer id="footer_container" class="fluid_container column-below-medium">
```

Comment:
is "fluid_container" a customized class? As I couldn't find it.
Maybe we can use bootstrap class "container-fluid" (if required)

I'm also not sure about the naming conventions of CSS classes but I guess it's all lowercase separated with '-'